src/__tests__/local-state/resolvers.ts

```diff
@@ -662,14 +678,16 @@ describe("Writing cache data from resolvers", () => {
     const client = new ApolloClient({
       cache: new InMemoryCache(),
       link: ApolloLink.empty(),
-      resolvers: {
-        Mutation: {
-          start(_data, _args, { cache }) {
-            cache.writeQuery({ query, data: { field: 1 } });
-            return { start: true };
+      localState: new LocalState({
+        resolvers: {
+          Mutation: {
+            start(_data, _args, { client }) {
+              client.cache.writeQuery({ query, data: { field: 1 } });
+              return { start: true };
```

Comment:
```suggestion
              return true;
```

We should be checking the result of `client.mutate` below since this value is wrong